src/VuetifyResource.vue

```diff
@@ -150,48 +150,50 @@
                     v-on:input="onSelectedChange"
                 >
                     <template slot="items" slot-scope="props">
-                        <td v-if="useCheckboxes">
-                            <v-checkbox
-                                hide-details
-                                primary
-                                v-model="props.selected"
-                            ></v-checkbox>
-                        </td>
-                        <td v-for="item in tableContent">
-                            <component
-                                :content="props.item[item.value]"
-                                :is="getColumnType(item.columnType)"
-                                :item="props.item"
-                                :table-column="item"
-                            ></component>
-                        </td>
-                        <td class="crud-actions">
-                            <v-tooltip left v-if="canUpdateResources([props.item])">
-                                <v-btn
-                                    color="green"
-                                    flat
-                                    icon
-                                    slot="activator"
-                                    v-on:click="openUpdateHandler(props.item[resourceKeyName])"
-                                >
-                                    <v-icon>$vuetify.icons.edit</v-icon>
-                                </v-btn>
-                                <span>{{ lang('update') }}</span>
-                            </v-tooltip>
-                            <v-tooltip left v-if="canDeleteResources([props.item])">
-                                <v-btn
-                                    color="red"
-                                    flat
-                                    icon
-                                    slot="activator"
-                                    @click="showDeleteConfirmation([props.item[resourceKeyName]])"
-                                >
-                                    <v-icon>$vuetify.icons.delete</v-icon>
-                                </v-btn>
-                                <span>{{ lang('delete') }}</span>
-                            </v-tooltip>
-                            <slot :resource="props.item" name="crudActionsAfter"></slot>
-                        </td>
+                        <tr @click="handleRowClick(props.item[resourceKeyName])">
```

Comment:
Hmm, ik weet niet hoe het zit met sommige browsers als het komt met een @click op een `tr` tag. Misschien dat `@click.native` wellicht beter is?

Reply:
Binding Native Events to Components
There may be times when you want to listen directly to a native event on the root element of a component. In these cases, you can use the .native modifier for v-on:

<base-input v-on:focus.native="onFocus"></base-input>
This can be useful sometimes, but it’s not a good idea when you’re trying to listen on a very specific element, like an <input>. For example, the <base-input> component above might refactor so that the root element is actually a <label> element: